docker/packager/packager

```diff
@@ -154,6 +170,7 @@ if __name__ == "__main__":
     parser.add_argument("--unbundled", action="store_true")
     parser.add_argument("--split-binary", action="store_true")
     parser.add_argument("--cache", choices=("", "ccache", "distcc"), default="")
+    parser.add_argument("--ccache_dir", default= os.getenv("HOME") + '/.ccache')
```

Comment:
Without `HOME` env variable we will get very vogue error. Let's add default: `os.getenv("HOME", "")`.